assets/components/dough_theme/form/_form.scss

```diff
@@ -85,10 +85,6 @@
   @extend %form-input;
 }
 
-.form__input-container .form__input:focus ~ .form__input-outline {
-  @extend %form-input-focus-container
```

Comment:
@Tr4pSt3R Line 85 extends a silent class that exists, in `/assets/helpers/_form-input.scss`. The error (or warning) occurs when the class you are extending doesn't exist.